azurecontainerapps/azurecontainerapps.ts

```diff
@@ -212,8 +317,38 @@ export class azurecontainerapps {
                 telemetryHelper.setImageScenario();
             }
 
-            // Create or update Azure Container App
-            appHelper.createOrUpdateContainerApp(containerAppName, resourceGroup, imageToDeploy, optionalCmdArgs);
+            // Create or update the Container App
+            if (!containerAppExists) {
```

Comment:
I noticed that this function was called `createOrUpdateContainerApp` before it was renamed. Should we consider still having a similar named function with this block of code, so that we can return early and reduce a bit the code nesting?

Reply:
I'll take a look at resolving this when refactoring/cleaning up this class along with the other refactoring comment you made above!